### PR TITLE
Update docs and signatures for Concordance

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

**Comment by tpoterba** [comment]
And mendel errors (which still needs to be rewritten in Python)

File: python/hail/methods/qc.py [hunk 1]
```diff
@@ -231,31 +231,33 @@ def concordance(left, right):
...
-        - **n_discordant** (:py:data:`.tfloat64`) -- Count of discordant calls (see below for
+        - `num_discordant` (:py:data:`.tfloat64`) -- Count of discordant calls (see below for
           full definition).
-        - **concordance** (:class:`.tarray` of :class:`.tarray` of :py:data:`.tint64`) --
+        - `concordance` (:class:`.tarray` of :class:`.tarray` of :py:data:`.tint64`) --
           Array of concordance per state on left and right, matching the structure of the
           global summary defined above.
 
-    In these tables, the column **n_discordant** is provided as a convenience,
+    In these tables, the column **num_discordant** is provided as a convenience,
```
Comment (jigold) [inline python/hail/methods/qc.py]: I thought everything was going to be `n_...`?

File: python/hail/methods/family_methods.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def mendel_errors(dataset, pedigree):
         - `fam_id` (:py:data:`.tstr`) -- Family ID.
         - `locus` (:class:`.tlocus`) -- Variant locus, key field.
         - `alleles` (:class:`.tarray` of :py:data:`.tstr`) -- Variant alleles, key field.
-        - `s` (:py:data:`.tstr`) -- Proband ID, key field.
+        - (column key of `dataset`) (:py:data:`.tstr`) -- Proband ID, key field.
```
Comment (jigold) [inline python/hail/methods/family_methods.py]: This is the second field of the struct. Move to below `fam_id`.

File: python/hail/methods/qc.py (hunk 1 above)
Comment (tpoterba) [reply python/hail/methods/qc.py]: this PR was made before that consensus, I'll update it

**Pull request merged**